scripts/build_pycer.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Rebuild pycer Python bindings
+# Forces a fresh build from C++ source
+
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)